changelog/dont-propagate-flags.dd

```diff
@@ -0,0 +1,10 @@
+Dont propagate DFLAGS and LFLAGS to sub commands by default
```

Comment:
The wording seems a bit off to me. "Don't ..." feels imperative and a little ambiguous. Something like "`DFLAGS` and `LFLAGS` are no longer propagated to [...]" sounds way more readable from a changelog PoV. Will edit myself.